code/__SANDCODE/DEFINES/misc.dm

```diff
@@ -1,2 +1,4 @@
 /// Adds an utf-8 header...? only ever used on circuitry so when wiremod arrives...
 #define UTF8HEADER "<meta http-equiv='Content-Type' content='text/html; charset=UTF-8' /><meta http-equiv='X-UA-Compatible' content='IE=edge' />"
+/// A shorthand for ternary operators to simulate a null-coalescing operator. Returns the first argument if its defined, otherwise, second.
+#define NULL_COALESCE(var, default) ((var) ? (var) : (default))
```

Comment:
Unnecessary, also incorrect.
What you're trying to implement here is `(isnull(var) ? var : default)` (your version gives the default on empty strings and the number 0, for example), but what you really want (and don't need a macro for) is `var || default`.

Reply:
Er, `isnull(var) ? default : var`.